test/test.js

```diff
@@ -1,12 +1,12 @@
 const sdk = require('../index.js');
 
-const liteApi = sdk('sand_c0155ab8-c683-4f26-8f94-b5e92c5797b9');
+const liteApi = sdk('sand_e7be24f5-f004-484c-9103-f4eba210b58f');
```

Comment:
What is this key? Please keep the one we have on the docs https://docs.liteapi.travel/reference/authentication